E2E/tests/avatarUpload.test.ts

```diff
@@ -0,0 +1,78 @@
+Feature('Avatar upload')
+
+const { I, login, game } = inject();
+import username = require("../test_data/usernames");
+import imageLink = require("../test_data/imageLinks");
+import ChessTile = require("../test_data/chessTiles");
+import { ChessPiece } from "../test_data/chessPieces";
+import { ChessPieceValue } from "../test_data/chessPieces";
+import assertions = require("../assertions/assertions");
+
+Scenario('Player uploads avatar during login', async () => { 
+  login.voterUploadAvatarDuringLogin(username.user1, imageLink.dogImageLink);
```

Comment:
there is some repetitive code. If it's easier for you, you can keep it this way, bet here is suggestion: You can create several folders in tests directory and separate tests a bit. Then you can use Before() hook regarding scenarios in each file

Reply:
A separate ticket will be created for that in the future.